funding/manager.go

```diff
@@ -3576,6 +3640,41 @@ func (f *Manager) newChanAnnouncement(localPubKey,
 		chanUpdateAnn.FeeRate = uint32(
 			ourPolicy.FeeProportionalMillionths,
 		)
+
+	case storedFwdingPolicy != nil:
+		chanUpdateAnn.BaseFee = uint32(storedFwdingPolicy.BaseFee)
+		chanUpdateAnn.FeeRate = uint32(storedFwdingPolicy.FeeRate)
+
+		// TODO: Decide between storing the defaults and using them here
```

Comment:
The question about the TODO is still open. Is this something that actually needs to be decided or can we remove the TODO instead?
Also, the lines following after the comment are now duplicated between this case and the default case. If we set the values when we initialize `chanUpdateAnn` and only overwrite what we can actually configure this `switch` statement becomes easier to read and understand IMO.